client/client.go

```diff
@@ -148,27 +155,13 @@ func (c *Client) Update() (data.TargetFiles, error) {
 }
 
 func (c *Client) update(latestRoot bool) (data.TargetFiles, error) {
-	// Always start the update using local metadata
-	if err := c.getLocalMeta(); err != nil {
+	if err := c.updateRoots(); err != nil {
```

Comment:
nit: as latestRoot bool is not used anymore `update` func can be removed in favor of Update()

Update func comment is also outdated